packages/devtools_app/lib/src/debugger/debugger_controller.dart

```diff
@@ -1057,6 +1112,37 @@ class DebuggerController extends DisposableController
 
     return positions;
   }
+
+  void toggleSearchInFileVisibility(bool visible) {
+    _showSearchInFileField.value = visible;
+    if (!visible) {
+      resetSearch();
+    }
+  }
+
+  @override
+  List<SourceToken> matchesForSearch(String search) {
```

Comment:
Can we adjust this to search across lines?

Reply:
you mean like search for a query like 'hello world' where 'hello' is on line 1 and 'world' is on line 2?

Reply:
The latter. `hello` on line 1 and `world` on line two. I thought Chrome supported this but that doesn't seem to be the case. This is probably fine regardless.

Reply:
The IDEs don't support this type of search either, so I think it's on par if we leave as is.